.env

```diff
@@ -65,3 +65,5 @@ SECTION_TITLE_CHAR_LIMIT=200
 
 PROJECT_REDIRECT_HASH_URL=https://s3.amazonaws.com/s3.documentcloud.org/legacy/project_redirects.bin
 ORG_REDIRECT_HASH_URL=https://s3.amazonaws.com/s3.documentcloud.org/legacy/org_redirects.bin
+
+export COMPOSE_FILE=local.yml
```

Comment:
I don't think this should be here - shouldn't this be in a local shell file?

Reply:
Where else does this file get used?